src/main/java/com/google/sps/servlets/GetGameDialogue.java

```diff
@@ -26,19 +31,26 @@
 public final class GetGameDialogue extends HttpServlet {
   private static final String TEXT_TO_HTML = "text/html;";
   private static final String JSON_CONTENT_TYPE = "application/json";
-  private static final String SOFTWAREID = "software-engineering-0";
-  private static final String TESTCONTENT = "hello this is a test";
-
-  // TODO: GameStage will change and be query from a database that will be implemented
-  private GameStage currentGameStage = new GameStage(SOFTWAREID, TESTCONTENT);
+  private static final String LOGGED_OUT_EXEPTION = "It appears the user you have is not logged in";
```

Comment:
typo, EXEPTION --> EXCEPTION

Message is a bit strange. Maybe, "It appears that you have not logged in."